From c431120daad830f77f7c23f6c78a2a5beeb8cfb6 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sat, 27 Jun 2020 12:20:42 +0800
Subject: [PATCH 091/104] arm64: allwinner: dts: a64: pinephone: lock DLDO4
 (VCC-WIFI-IO) to 1v8

Because some IO pins of the Wi-Fi chip is connected to PL bank (which is
set to 1v8) instead of PG bank, the VCC-WIFI-IO voltage must be set to
1v8.

Restrict the range of DLDO4 to 1v8, to prevent MMC subsystem from selecting
3v3 IO for SDIO port.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 82584a9157717..6e2b1c98bf4e0 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -491,7 +491,7 @@ &reg_dldo3 {
 
 &reg_dldo4 {
 	regulator-min-microvolt = <1800000>;
-	regulator-max-microvolt = <3300000>;
+	regulator-max-microvolt = <1800000>;
 	regulator-name = "vcc-wifi-io";
 };
 
-- 
2.26.2

